ethers-middleware/tests/it/gas_escalator.rs

```diff
@@ -0,0 +1,41 @@
+use ethers_core::{types::*, utils::Anvil};
+use ethers_middleware::{
+    gas_escalator::{Frequency, GasEscalatorMiddleware, GeometricGasPrice},
+    MiddlewareBuilder,
+};
+use ethers_providers::{Http, Middleware, Provider};
+use ethers_signers::{LocalWallet, Signer};
+
+#[tokio::test]
+#[ignore]
```

Comment:
I don't know how gas escalator works, I tried porting the test to anvil but it was broken before too. I can't replicate the escalation

Reply:
maybe the assertions or the gas prices are wrong which I added, everything before that is the same just rewritten using anvil